orchagent/portsorch.cpp

```diff
@@ -805,10 +862,7 @@ void PortsOrch::doVlanMemberTask(Consumer &consumer)
                 continue;
             }
 
-            /* Assert the port doesn't belong to any VLAN */
-            assert(!port.m_vlan_id && !port.m_vlan_member_id);
-
-            if (addBridgePort(port) && addVlanMember(vlan, port, tagging_mode))
+            if ((port.m_bridge_port_id || addBridgePort(port)) && addVlanMember(vlan, port, tagging_mode))
```

Comment:
>port.m_bridge_port_id ||  [](start = 17, length = 25)

since you check this in addBridgePort, you do not need to check the bridge_port_id here.

Reply:
As explained before.